src/main/java/org/swisspush/reststorage/EventBusAdapter.java

```diff
@@ -372,14 +426,34 @@ public HttpServerResponse writeCustomFrame(int type, int flags, Buffer payload)
                         throw new UnsupportedOperationException();
                     }
 
+                    @Override
+                    public HttpServerResponse addCookie(Cookie cookie) {
+                        return null;

```

Comment:
We should `throw new UnsupportedOperationException();` for those methods instead of returning `null`